common/logger/logger.c

```diff
@@ -319,3 +318,9 @@ log_read_e_t get_log_read_status()
 {
     return sg_log_data.read_sm_e;
 }
+
+void lv_custom_log_handler(lv_log_level_t level, const char *file, int line, const char *dsc) {
```

Comment:
the function prototype is wrong. Correction: 
```suggestion
void lv_custom_log_handler(lv_log_level_t level, const char *file, uint32_t line, const char *dsc) {
```